### ISSUE TITLE
Dryad-dev is no longer serving most pages through Apache (which rewrites or handles SSL)

### ISSUE THREAD

**Issue by sfisher** [CDL-Dryad/dryad-product-roadmap#2326]
Bad Gateway
The proxy server received an invalid response from an upstream server.

However curl for the server seems to work like `curl http://localhost:3000/stash` from within the same machine.

Some (static, I think) pages work like
- https://dryad-dev.cdlib.org/api/v2/
- https://dryad-dev.cdlib.org/api/v2/docs/

But these are maybe either static pages or serving non-html.

Maybe there is some problem with the mime types or headers or something else?  Stage is working.

This happened after Ashley applied security updates and rebooted.

Nagios is disabled right now from alerting until we can fix.

**Comment by sfisher** [comment]
I tracked this down to the `link` header which is really big on development since it lists every single js/css file and doesn't compile them into one big one to simplify debugging.

I tested the change below in apache and it worked until puppet redeployed again.  The only part of the line that changed was the `responsefieldsize` on the end.

```
BalancerMember http://localhost:3000 max=64 acquire=10 timeout=600 Keepalive=On responsefieldsize=32768
```

Ashley is updating puppet for me.  Thank you, Ashley.

I finally looked at the correct log and figured out the problem. ;-)
